lxc/utils.go

```diff
@@ -315,3 +316,72 @@ func getServerSupportedFilters(filters []string, i interface{}) ([]string, []str
 
 	return supportedFilters, unsupportedFilters
 }
+
+func guessImage(conf *config.Config, d lxd.InstanceServer, remote string, iremote string, image string) (string, string) {
+	if remote != iremote {
+		return iremote, image
+	}
+
+	fields := strings.SplitN(image, "/", 2)
+	_, ok := conf.Remotes[fields[0]]
+	if !ok {
+		return iremote, image
+	}
+
+	_, _, err := d.GetImageAlias(image)
+	if err == nil {
+		return iremote, image
+	}
+
+	_, _, err = d.GetImage(image)
+	if err == nil {
+		return iremote, image
+	}
+
+	if len(fields) == 1 {
+		fmt.Fprintf(os.Stderr, i18n.G("The local image '%s' couldn't be found, trying '%s:' instead.")+"\n", image, fields[0])
```

Comment:
When you come across `'%s'` in a general message to the user, please can you change them to `%q`, as we are using that for quoting consistency now.